<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>710</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>710</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>710</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>850</width>
    <height>710</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gestione Personale</string>
  </property>
  <widget class="QPushButton" name="atletiButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>139</width>
     <height>91</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200,200,200)</string>
   </property>
   <property name="text">
    <string>ATLETI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="istruttoriButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>139</width>
     <height>91</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200,200,200)</string>
   </property>
   <property name="text">
    <string>ISTRUTTORI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="amministratoriButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>40</y>
     <width>139</width>
     <height>91</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200,200,200)</string>
   </property>
   <property name="text">
    <string>AMMINISTRATORI</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="displayTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>771</width>
     <height>401</height>
    </rect>
   </property>
   <property name="alternatingRowColors">
    <bool>false</bool>
   </property>
   <column>
    <property name="text">
     <string>Nome</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Cognome</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Data di Nascita</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Username</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Password</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tipologia di Utente</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Email</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Telefono</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="creaUtenteButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>590</y>
     <width>139</width>
     <height>91</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(140,255,70)</string>
   </property>
   <property name="text">
    <string>CREA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modifyUtenteButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>590</y>
     <width>139</width>
     <height>91</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255,255,70)</string>
   </property>
   <property name="text">
    <string>MODIFICA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>590</y>
     <width>139</width>
     <height>91</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255,70,70)</string>
   </property>
   <property name="text">
    <string>INDIETRO</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
